.github/workflows/sync-upstream.yaml

```
@@ -20,7 +20,7 @@ jobs:
         uses: autowarefoundation/autoware-github-actions/sync-branches@v1
         with:
           token: ${{ steps.generate-token.outputs.token }}
-          base-branch: main
+          base-branch: tier4/universe
```

No tier4/universe branch? :thinking: 

```suggestion
          base-branch: main
```

---

In https://github.com/tier4/autoware_individual_params.s1, I use tier4/universe as main.
However, [jpntaxi](https://github.com/tier4/autoware_individual_params.jpntaxi) repository seems to not have tier4/universe but main.
I'll change to main branch in S1 repository